src/functions/types.ts

```diff
@@ -396,3 +397,32 @@ export type RuntimeUnhandledEventContext<
     // deno-lint-ignore no-explicit-any
     body: any;
   };
+
+type DefineFunctionReturnType = ReturnType<typeof DefineFunction>;
+
+type FunctionRuntimeReturnArgs<
+  FunctionReturnType extends
+    | FunctionHandlerReturnArgs<UnknownRuntimeType>
```

Comment:
Just to confirm / my own education, use of `UnknownRuntimeType` here is only to avoid the `any` warnings from deno, yes?

Reply:
Similar to my comment above, we could directly use the expected shape instead.

```ts
FunctionHandlerReturnArgs<FunctionParameters>
```

Reply:
I've improved the types, this isn't `AnyScript`

Reply:
TBH I love anyscript